add content to user manual

LimEunSang opened the pull request:
I made a simple instructions. If you give me any feedback, I'll work more.
I'd appreciate it if you could give me any suggestions.

LimEunSang committed:
add instructions.md

LimEunSang committed:
add Instructions rink in User manual

LimEunSang committed:
Create collection.md

LimEunSang committed:
Update SUMMARY.md

nukeop commented on user-manual/collection.md:
```diff
@@ -0,0 +1,37 @@
+
+You can add or delete albums to the list by clicking the star button.
+
+![Add Favorite Albums](https://i.imgur.com/xVp5iFO.png)
```
Thanks for taking the screenshots, could you please make sure they're in their original size, so they're not blurry? And please crop them to the exact size of the window so there are no white borders.

LimEunSang replied:
I tried to modify it according to your feedback. The pixels in the current picture are 1385 x 918. If the picture is still blurry or broken, can you tell me the ideal pixel?

nukeop replied:
Thanks, it looks great now. Would you like to add anything else, or is it ready to merge?

LimEunSang replied:
I want to work more!

nukeop commented:
Thanks for this contribution, I left a comment with a possible improvement. I'll probably want to edit it a little to improve grammar etc, but I'll do that after merging.

LimEunSang committed:
change pictures and modify picture's name

I removed the blurring and made the white bounders invisible.

LimEunSang committed:
add pictures and add some categories

add  pictures to explain it in detail.
add Equalizer, Visualizer, and Language categories

LimEunSang committed:
modify Importing playlists Link

LimEunSang committed:
Create plugins.md

Write the list of  stream, metadata, lyrics providers and plugin API link

LimEunSang committed:
add Local Library

LimEunSang committed:
One more line to Metadata providers.

LimEunSang commented:
I've completed the task. I'll wait for your feedback or merge.

LimEunSang committed:
add plugins.md link in user manual

and change the order of arrangement. (Instructions - Collection - Importing playlists - Plugins - Keyboard shortcuts)
